app/upgrades/v22/upgrades.go

```diff
@@ -23,6 +23,9 @@ func CreateUpgradeHandler(
 			return nil, err
 		}
 
+		// For sake of simplicity, we restart the taker fee tracker accounting height.
+		keepers.PoolManagerKeeper.SetTakerFeeTrackerStartHeight(ctx, ctx.BlockHeight())
```

Comment:
Shouldn't we be keeping continuinity? THis is going to be much harder for every integrator for the future if we don't, no?

Reply:
Implemented here https://github.com/osmosis-labs/osmosis/pull/7240/commits/f19b8194b3ab15a4d6df4f2272159f613198d606

@ValarDragon would appreciate an ack on the implementation so it doesn't get swept under the rug when merging